src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +10,18 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private static final UserService userService = new UserService();
+
     public boolean login(String email, String password) {
+        if (email == null || password == null) {
+            return false;
+        }
+
```

Comment:
validation is most probably done on previous stages, assume it's out of scope of the task
```suggestion
```